utils/markdown/blocks.go

```diff
@@ -77,10 +77,10 @@ func ParseBlocks(markdown string, lines []Line) (*Document, []*ReferenceDefiniti
 				didAdd := false
 				for i := lastMatchIndex; i >= 0; i-- {
 					if container, ok := openBlocks[i].(ContainerBlock); ok {
-						if newBlocks := container.AddChild(newBlocks); newBlocks != nil {
+						if addedBlocks := container.AddChild(newBlocks); addedBlocks != nil {
```

Comment:
I think is correct.

Reply:
Just to confirm my understanding: the `newBlocks` in the old code was a new variable, given the scope of the conditional. So renaming it is perfectly safe, since we could never have referred to the value in the outside context.